dependencies.yaml

```diff
@@ -245,32 +245,33 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - GPUtil>=1.4.0
-          - imagecodecs>=2021.6.8
-          - openslide-python>=1.1.2
           - psutil>=5.8.0
           - pytest-cov>=2.12.1
           - pytest-lazy-fixture>=0.6.3
           - pytest-xdist
           - pytest>=6.2.4
           - tifffile>=2022.7.28
+          - pooch>=1.6.0  # needed to download scikit-image sample data
       - output_types: [conda]
         packages:
+          - imagecodecs>=2021.6.8
+          - matplotlib-base
+          - openslide-python>=1.1.2
```

Comment:
Like `imagecodecs` above, it seems like `openslide-python` is pinned the same way for `conda` and `requirements`. Do you want to list that above in the `[conda, requirements, pyproject]` section?

Note that in most cases you want the same dependencies in `requirements` and `pyproject` because they both install from PyPI, so I'm questioning why we have both `[requirements]` and `[requirements, pyproject]` sections. I expect we really only need three sections: `[conda, requirements, pyproject]` for dependencies pinned the same way in conda/pip, `[conda]` for conda-only dependencies, and `[requirements, pyproject]` for pip-only dependencies.

Reply:
removed duplicate definitions. Install of `imagecodecs` and `openslide-python` is currently disabled at wheel test time, but we intend to fix that after this PR is merged (opened issue #626 to track)